forge-gui/res/cardsfolder/upcoming/the_first_tyrannic_war.txt

```diff
@@ -0,0 +1,11 @@
+Name:The First Tyrannic War
+ManaCost:2 G U R
+Types:Enchantment Saga
+K:Saga:3:DBChangeZone,DBDouble,DBDouble
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | SubAbility$ DBCounter | ChangeType$ Creature.YouCtrl | Optional$ True | RememberChanged$ True | SpellDescription$ You may put a creature card from your hand onto the battlefield. If its mana cost contains {X}, it enters the battlefield with a number of +1/+1 counters on it equal to the number of lands you control.
+SVar:DBCounter:DB$ PutCounter | Defined$ Remembered | IsPresent$ Creature.IsRemembered+hasXCost | CounterType$ P1P1 | CounterNum$ X |
```

Comment:
It should ETB with the counters, might need more support

Reply:
Why resolve?
It matters for interaction like "Whenever a creature with power X or greater enters the battlefield"

Reply:
Oh I understand now, I'll just create a new PR with it later.